### DISCUSSION TITLE
recipes/ingredients/pastry-dough/deluxe-butter-pastry/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#859] [Comments]
# recipes/ingredients/pastry-dough/deluxe-butter-pastry/

A collection of my recipes.

https://nicholaswilde.io/recipes/ingredients/pastry-dough/deluxe-butter-pastry/

**Comment by nicholaswilde** [comment]
Parbake 20-25 minutes at 400 with pie weights.
Bake again at 325 after removing weights for 20 minutes until bottom is golden brown.
